packages/next/build/webpack-config.ts

```diff
@@ -1344,6 +1344,10 @@ export default async function getBaseWebpackConfig(
         'process.env.__NEXT_OPTIMIZE_CSS': JSON.stringify(
           config.experimental.optimizeCss && !dev
         ),
+        'process.env.__NEXT_OPTIMIZE_SCRIPTS': JSON.stringify({
+          enablePartytown: config.experimental.optimizeScripts?.enablePartytown,
+          partytownConfig: config.experimental.optimizeScripts?.partytownConfig,
```

Comment:
Should we make this a single config? Something like `workerOptimization`. This is similar to our other options

Reply:
Updated to a single boolean!

I kept the name to be `optimizeScripts` because I'm hoping to eventually bundle other experimental script functionality with the same flag. It also matches the other names (`optimizeCss`, `optimizeFonts`) but let me know if you still think I should change the name to something like `workerOptimization`.